pipeline-braille-scripts/xml-to-pef/src/main/README.md

```diff
@@ -0,0 +1,12 @@
+# xml-to-pef API
+
+## <a href="resources/META-INF/catalog.xml" class="source">catalog.xml</a>
+
+## OSGi services
+
+### Scripts (`org.daisy.pipeline.script.XProcScriptService`)
+
+- [`http://www.daisy.org/pipeline/modules/braille/xml-to-pef/xml-to-pef.xpl`](resources/xml/xproc/xml-to-pef.xpl)
```

Comment:
I think the catalog.xml part is still relevant. But xml-to-pef.xpl is not a org.daisy.pipeline.script.XProcScriptService.